astropy/io/fits/fitstime.py

```diff
@@ -0,0 +1,251 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import re
+import warnings
+from collections import defaultdict, OrderedDict
+
+import numpy as np
+
+from . import Header, Card
+
+from ...coordinates import EarthLocation
+from ...table import Column
+from ...time import Time
```

Comment:
@AustereCuriosity - Yes, at least that's the general idea. But it may need some testing to check taht everything is ok (like testing import times).